README.md

```diff
@@ -53,18 +53,39 @@ Now that we've covered the basics of using `dotnew new`, lets move on to info fo
 If you're authoring templates, or interested in contributing to this repo, then you're likely interested in how to use the latest version of this experience.
 The steps required are outlined below.
 
-Step 1: Get the SDK for your platform from [dotnet/cli](https://github.com/dotnet/cli)
+## Prerequisites
+In order to work with this project, you will need the following.
+- [Visual Studio 2017](https://www.visualstudio.com/)
```

Comment:
The build that produces the template NuGet packages currently only works on Windows (as it depends on the full framework flavor of the NuGet client). This was a larger problem before #401 because setup attempted to install those NuGet packages. Setup now installs the templates from source (in the `template_feed` directory), so there's not a build from source for personal use impact.

For users wishing to install using the template NuGet packages - they'll either need to be on Windows (to be able to produce the packages) or install them from the MyGet feed (which they'd need an appropriate NuGet config).